openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +492,37 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is offered at the Request URI endpoint by the Verifier. In case of success, the response is a request object that the Wallet MUST process in the same way as a request object as defined in [@RFC9101]. 
+
+The request MUST use the POST method, the https scheme and the media type set to "application/oauth-authz-req+jwt".
+
+The following parameters are defined: 
+
+`state`:
+: OPTIONAL. A JSON String containing the value of the corresponding authorization Request's `state` parameter, if present.
+
+`issuer`:
```

Comment:
This seems duplicate to `wallet_metadata` and may add more difficulty for the wallet if it has to merge the metadata from two possiblities. I'm in favor of only transmitting the ata with `wallet_metadata`

Reply:
Those are two options to pass metadata. 
wallet_metadata allows to send dynamically created data, issuer has the advantage of giving some binding between the issuer URL and the actual metadata. I think both are valid options (until we have substantial implementation experience).